nubi/src/components/home/home.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import nubi from '../../assets/images/nubi-paypal-logo.svg';
+import phone from '../../assets/images/img_main_section_phone@2x.png'
+import RegisterButton from '../buttons/register/button';
+import HomeTwo from './homeTwo';
+import HomeThree from './homeThree';
+
+class Home extends React.Component {
+  render(){
+  return(
```

Comment:
Ten en cuenta que usar los espacios adecuadamente mejora la legibilidad del código: 
```suggestion
  render() {
  return (
```